src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+          } = person;
+
+          const mother = people.find(parent => parent.name === motherName);
+          const father = people.find(parent => parent.name === fatherName);
+
+          const mum = mother && people.includes(mother)
```

Comment:
There is no need to check if people array includes mother as you already found it, so it definitely includes